src/core/stability.py

```
@@ -0,0 +1,116 @@
+"""Screen stability detection module."""
+
+from typing import Optional, List
+import numpy as np
+
+from src.utils.image import calculate_ssim, get_image_brightness, is_dark_mode
```

Ruff (0.12.2)

`src.utils.image.get_image_brightness` imported but unused

Remove unused import: `src.utils.image.get_image_brightness`

(F401)